### ISSUE TITLE
Can't Update Description in route

### ISSUE THREAD

**Issue by andredeo** [OpenVPN/terraform-provider-openvpn-cloud#34]
# Problem
The module fails in update OpenVPN data, but **update the terraform state**.

Compare failed update:
```~ description     = "Testa" -> "Alert Manager 0 - IP 2"```
With the state:
```description     = "Alert Manager 0 - IP 2"```

# Create Object
```
$ terraform apply

An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  # openvpncloud_route.route will be created
  + resource "openvpncloud_route" "route" {
      + description     = "Testa"
      + id              = (known after apply)
      + network_item_id = "73a46ae8-8c61-43a7-9708-ab101cac1ac5"
      + type            = "IP_V4"
      + value           = "55.230.236.91/32"
    }

Plan: 1 to add, 0 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

openvpncloud_route.route: Creating...
openvpncloud_route.route: Creation complete after 0s [id=50ca39a5-1535-40aa-9701-c7c61ea8b745]

Apply complete! Resources: 1 added, 0 changed, 0 destroyed.
```

# Update Object
```
$ terraform apply
openvpncloud_route.route: Refreshing state... [id=50ca39a5-1535-40aa-9701-c7c61ea8b745]

An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

  # openvpncloud_route.route will be updated in-place
  ~ resource "openvpncloud_route" "route" {
      ~ description     = "Testa" -> "Alert Manager 0 - IP 2"
        id              = "20cf39a2-1232-40af-9701-c7d61ea8b742"
        network_item_id = "73a46ae8-8c61-43a7-9708-ab101cac1ac5"
        type            = "IP_V4"
        value           = "55.230.236.91/32"
    }

Plan: 0 to add, 1 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

openvpncloud_route.route: Modifying... [id=50ca39a5-1535-40aa-9701-c7c61ea8b745]

Error: Status code: 400, Response body: {"path":"/api/beta/networks/73f46ae8-8c61-43a7-9708-fb101dad1fc2/routes","requestId":"21571ffa-803","status":400,"statusError":"Bad Request","timestamp":1678134794205}

  on main.tf line 20, in resource "openvpncloud_route" "route":
  20: resource "openvpncloud_route" "route" {
```

# List Object State
```
terraform  state show openvpncloud_route.route
# openvpncloud_route.route:
resource "openvpncloud_route" "route" {
    description     = "Alert Manager 0 - IP 2"
    id              = "20cf39a2-1232-40af-9701-c7d61ea8b742"
    network_item_id = "73a46ae8-8c61-43a7-9708-ab101cac1ac5"
    type            = "IP_V4"
    value           = "55.230.236.91/32"
}
```

**Comment by vladimir-kozyrev** [comment]
I have looked into this and found that the update of the route triggers its creation.
Since it already exists, we get 400 Bad Request in response.

I will work on a fix.

**Comment by vladimir-kozyrev** [comment]
Should be fixed now since the release of v0.0.9.
